walletunlocker/service.go

```diff
@@ -652,8 +652,8 @@ func (u *UnlockerService) ChangePassword(ctx context.Context,
 
 // ValidatePassword assures the password meets all of our constraints.
 func ValidatePassword(password []byte) error {
-	// Passwords should have a length of at least 8 characters.
-	if len(password) < 8 {
+	// Passwords should have a length of at least 8 characters, but allow default password.
+	if len(password) < 8 && string(password) != "hello" {
```

Comment:
I would use `lnwallet.DefaultPublicPassPhrase` or `lnwallet.DefaultPrivatePhrase` here to avoid the hardcoded string. A probably better solution is to avoid the `ValidatePassword` check when we know it's changing to the defaults?

Reply:
Done